server/resolvers/BrowserVersionOperations/editBrowserVersionResolver.js

```diff
@@ -0,0 +1,25 @@
+const { AuthenticationError } = require('apollo-server');
+const {
+    updateBrowserVersionById
+} = require('../../models/services/BrowserService');
+
+const editBrowserVersionResolver = async (
+    { parentContext: { id } },
+    { updatedName },
+    { user }
+) => {
+    if (
+        !(
+            (
+                user?.roles.find(role => role.name === 'ADMIN') ||
+                user?.roles.find(role => role.name === 'TESTER')
+            ) // Tester is allowed to change Browser Version when it cannot be automatically detected
+        )
```

Comment:
This could be something like `if (user === null) {`! I don't think a comment is needed since the code is pretty clear.